govc/README.md

```diff
@@ -1,39 +1,45 @@
 # govc
 
-govc is a vSphere CLI built on top of govmomi.
+`govc` is a vSphere CLI built on top of govmomi.
 
 The CLI is designed to be a user friendly CLI alternative to the GUI and well suited for automation tasks.
 It also acts as a [test harness](test) for the govmomi APIs and provides working examples of how to use the APIs.
 
 ## Installation
 
-You can find prebuilt govc binaries on the [releases page](https://github.com/vmware/govmomi/releases).
+### Docker
+
+The official `govc` [Docker images](https://hub.docker.com/r/vmware/govc) are built from this [Dockerfile](../Dockerfile.govc).
+
+### Binaries
+You can find prebuilt `govc` binaries on the [releases page](https://github.com/vmware/govmomi/releases).
 
 Download and install a binary locally like this:
 
 ``` console
-% curl -L $URL_TO_BINARY | gunzip > /usr/local/bin/govc
-% chmod +x /usr/local/bin/govc
+$ curl -L $URL_TO_BINARY | gunzip > /usr/local/bin/govc
```

Comment:
Ah, we need to change this one, now that the releases are `.tar.gz`

Reply:
Will do too!